tensilelite/Tensile/Common.py

```diff
@@ -1609,6 +1585,39 @@ def assignGlobalParameters( config ):
 
   global globalParameters
 
+  if config["ValidateMatrixInstructions"]:
```

Comment:
A comment here clarifying that this is set to True by default when executing ./Tensile but set to False when executing ./TensileCreateLibrary would be helpful for future developers to understand the context of this change.

Reply:
Is there a way to pass the same object to all working process instead of turning off? If this is turned off, there is no way to check the correctness of the logic yaml files.

Reply:
@KKyang There are options in joblib to allow for the use of shared memory, but we haven't conducted too much research into this, and from what we found it was error prone and brittle. The intention here is that the validation should already be conducted prior to saving this as a long-term logic file, thus, the step shouldn't be required in TensileCreateLibrary. While keeping this validation does provide some peace of mind, it's also one of the largest contributors to memory consumption, so we feel the trade of is worth it.

Reply:
@KKyang this is off by default in the context of TensileCreateLibrary because I couldn't find where the check was applied. However, it is on by default for Tensile which in that context the check is applied.